pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -775,6 +775,8 @@ let
       X86_CHECK_BIOS_CORRUPTION = yes;
       X86_MCE                   = yes;
 
+      RAS = yes;
```

Comment:
Maybe add a comment stating why this is needed?

Reply:
Sure I suppose I can do that. Though it might be more natural to explicitly enable EDAC as well (as building that really is the intent here) even though the current change is sufficient.